apps/storefront/src/pages/quote/components/QuoteTable.tsx

```diff
@@ -251,13 +251,27 @@ function QuoteTable(props: ShoppingDetailTableProps) {
               warningMessage = b3Lang('quoteDraft.quoteTable.unavailable.tip');
             }
           }
-        } else if (!product.unlimitedBackorder) {
-          const productStock = isInventoryTrackingEnabled ? product.availableToSell : row.quantity;
-          if (isInventoryTrackingEnabled && productStock < row.quantity) {
-            warningMessage = b3Lang('quoteDraft.quoteTable.outOfStock.tip');
-            warningDetails = b3Lang('quoteDraft.quoteTable.oosNumber.tip', {
-              qty: productStock,
-            });
+        } else {
+          let hasUnlimitedBackorder = product.unlimitedBackorder;
+
+          if (isInventoryTrackingEnabled && product.inventoryTracking === 'variant') {
```

Comment:
can you lift the check for `isInventoryTrackingEnabled`, it looks like all the conditions in this if depend on it being `true`.